config/features.yml

```diff
@@ -2395,4 +2395,8 @@ features:
   gi_comparison_tool_cautionary_info_update:
     actor_type: user
     description: Enables updated cautionary info student feedback ui for GI Comparison Tool
+  vaos_add_substance_use_disorder:
```

Comment:
On second thought, we usually use the `va_online_scheduling_` prefix. I don't mind switching to a new prefix but we might want to discuss this after standup one of these days. It is kinda overdue since we are now UAE/Orion so maybe va_online_scheduling/vaos are outdated terms.

Reply:
uae is nice and short and easily findable/differentiable (probably) but do others know what it means?